backend/corpora/common/utils/corpora_constants.py

```diff
@@ -1,4 +1,35 @@
+import collections
+
+
 class CorporaConstants(object):
+    # Namedtuples used to encapsulate information about a piece of metadata for verification purposes
+    TypedMetadata = collections.namedtuple(
+        "TypedMetadata", "field_name required_type valid_alternative", defaults=[None]
+    )
+    Ontology = collections.namedtuple("Ontology", "ontology_name s3_uri")
+
+    # Constants related to type checking ontology fields
+    EFO_ONTOLOGY = Ontology(ontology_name="efo", s3_uri="corpora-data-portal-ontologies/efo.txt")
```

Comment:
Do you have the code to make all of these? We'll probably have to update them occasionally.

Reply:
Yeah I added my script to script directory.